helm/hpcc/templates/_helpers.tpl

```diff
@@ -313,6 +313,30 @@ Add Secret volume for a component
 {{- end -}}
 {{- end -}}
 
+{{/*
+Add sentinel-based probes for a component
+*/}}
+{{- define "hpcc.addSentinelProbes" -}}
+env:
+- name: "SENTINEL"
+  value: "{{ .name }}.sentinel"
+startupProbe:
+  exec:
+    command:
+    - cat
+    - "{{ .name }}.sentinel"
+  initialDelaySeconds: 5
+  failureThreshold: 30
```

Comment:
this could be too low for Dali startup with a big store.

When does it start probing? Is it only once the pod has reported as started?

Reply:
The startup probe will (I assume) start when the pod is marked as "running" 30 here means 30 retries, so timeout is actually 30*5

We may as you say want to make it possible for us - or helm user - to override for different components